ui/appui-react/src/appui-react/preview/PreviewFeatures.ts

```diff
@@ -79,14 +70,25 @@ export function trimToKnownFeaturesOnly(previewFeatures: PreviewFeatures) {
   return knownFeatures;
 }
 
-/**
- * Hook to access the preview features set in the Redux store.
+interface PreviewFeaturesState {
+  previewFeatures: PreviewFeatures;
+  setPreviewFeatures: (previewFeatures: PreviewFeatures) => void;
+}
+
+/** @internal */
```

Comment:
Added

Reply:
Ok, the goal was for us to be aware that these can also be used outside of react hooks though. Because unless you do a code search, or read the zustand documentation, it is not clear that it is an option.

Speaking of which, should we use this same approach for the `appui-layout-react` version of this ? As I mention in https://github.com/iTwin/appui/pull/601/files/55a24c3d6f57aa78dcc0b61d93da6d7f8a9c3077#r1404540889 I feel that this would be much more useful for case in this PR, where we can have access to know if the preview feature is on while in the NineZoneStateReducer.

Reply:
As discussed, let's keep a single preview features store and pass down what's needed via props.